lib/models/environment-variables.coffee

```diff
@@ -158,22 +159,23 @@ exports.device = {}
 # @public
 # @function
 #
-# @param {(String|Number)} deviceId - device id
+# @param {String} deviceName - device name
 # @param {module:resin.models.environment-variables.device~getAllCallback} callback - callback
 #
 # @example
-#	resin.models.environmentVariables.device.getAll 51, (error, environmentVariables) ->
+#	resin.models.environmentVariables.device.getAll 'MyDevice', (error, environmentVariables) ->
 #		throw error if error?
 #		console.log(environmentVariables)
 ###
-exports.device.getAll = (deviceId, callback) ->
-	return pine.get
-		resource: 'device_environment_variable'
-		options:
-			filter:
-				device: deviceId
-			expand: 'device'
-			orderby: 'env_var_name asc'
+exports.device.getAll = (deviceName, callback) ->
+	deviceModel.get(deviceName).then (device) ->
```

Comment:
I was thinking of an optional array parameter to specify the files you want, ie:
```coffee
deviceModel.get(deviceName, ['id'])
```
```coffee
deviceModel.get(deviceName, ['id', 'name])
```
or potentially:
```coffee
deviceModel.get(deviceName, 'id')
```
```coffee
deviceModel.get(deviceName, 'id', 'name')
```
however whilst the second version is nicer to type, it would be harder to distinguish since you already have potentially multiple parameters like that for filtering right?  
Could also maybe do it via an options object (which would be usable for other things as well in the future):
```coffee
deviceModel.get(deviceName, select: 'id')
```
```coffee
deviceModel.get(deviceName, select: ['id', 'name'])
```